### PR TITLE
Add Deneyap Color Converter Arduino Library

### PR DESCRIPTION
**Pull request by DogushC**
Add Deneyap Color Converter Arduino Library

### CODE REVIEW

File: repositories.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ https://github.com/ftjuh/RCWL_1X05
 https://github.com/UlmerMan/Elegoo
 https://github.com/HakkanR/slowAES
 https://github.com/cubicleguy/su_arduino_spi
-https://github.com/cubicleguy/su_arduino_uart 
+https://github.com/cubicleguy/su_arduino_uart
```
Comment (per1234) [inline repositories.txt:23]:
```suggestion
https://github.com/cubicleguy/su_arduino_uart 
```

File: repositories.txt [hunk 2]
```diff
@@ -5610,7 +5610,7 @@ https://github.com/EmotiBit/EmotiBit_EmojiLib
 https://github.com/d10i/TFA433
 https://github.com/IncroyablePix/DigiKeyboardBe
 https://github.com/ClayXrex/Arduino-StepperMotor
-https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library 
+https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library
```
Comment (per1234) [inline repositories.txt:5613]:
```suggestion
https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library 
```

**Review by per1234 — CHANGES_REQUESTED** [review]
@DogushC please revert the whitespace changes you made on lines 23 and 5613 of `repositories.txt`. You are very welcome to submit a separate pull request to clean up the trailing whitespace, but a submission PR must consist only of adding your repository URL to the list.

Any other changes confuse the automated system.

**Comment by DogushC** [comment]
> @DogushC please revert the whitespace changes you made on lines 23 and 5613 of `repositories.txt`. You are very welcome to submit a separate pull request to clean up the trailing whitespace, but a submission PR must consist only of adding your repository URL to the list.
> 
> Any other changes confuse the automated system.

Thanks. I committed these.

**Comment by per1234** [comment]
Thanks! A merge conflict resulted from your fork becoming outdated in the meantime. If you will fix that merge conflict, the submission will be automatically accepted.

You can learn how to do that via the GitHub web interface here:

https://docs.github.com/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

File: repositories.txt [hunk 3]
```diff
@@ -21,7 +21,7 @@ https://github.com/ftjuh/RCWL_1X05
 https://github.com/UlmerMan/Elegoo
 https://github.com/HakkanR/slowAES
 https://github.com/cubicleguy/su_arduino_spi
-https://github.com/cubicleguy/su_arduino_uart 
+https://github.com/cubicleguy/su_arduino_uart
```
Comment (per1234) [inline repositories.txt:24]:
```suggestion
https://github.com/cubicleguy/su_arduino_uart 
```

File: repositories.txt [hunk 4]
```diff
@@ -5611,7 +5611,7 @@ https://github.com/EmotiBit/EmotiBit_EmojiLib
 https://github.com/d10i/TFA433
 https://github.com/IncroyablePix/DigiKeyboardBe
 https://github.com/ClayXrex/Arduino-StepperMotor
-https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library 
+https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library
```
Comment (per1234) [inline repositories.txt:5614]:
```suggestion
https://github.com/jpconstantineau/BlueMicro_Examples_Arduino_Library 
```

**Review by per1234 — CHANGES_REQUESTED** [review]
Same as before. Please revert the whitespace changes so the automated system can correctly process you submission.

**Comment by DogushC** [comment]
> Same as before. Please revert the whitespace changes so the automated system can correctly process you submission.

I did these.

**Review by per1234 — APPROVED** [review]

**Comment by per1234** [comment]
@ArduinoBot please check it again

**Pull request merged**